test/helpers/transactions/InvokeScriptTransactionScreen.ts

```diff
@@ -0,0 +1,53 @@
+const Argument = (wrapped: WebdriverIO.Element) => ({
+  get type() {
+    return wrapped.findByTestId$('invokeArgumentType');
+  },
+
+  get value() {
+    return wrapped.findByTestId$('invokeArgumentValue');
+  },
+});
+
+const Payment = (wrapped: WebdriverIO.Element) => ({
```

Comment:
Is this wrapper needed? It just wraps single element

Reply:
done